python/demo/demo_lagrange_variants.py

```diff
@@ -19,23 +19,29 @@
 
 # +
 import numpy as np
-import ufl
 
+import ufl
 from dolfinx import fem, mesh
 from ufl import ds, dx, grad, inner
 
 from mpi4py import MPI
 from petsc4py.PETSc import ScalarType
 
+if np.issubdtype(ScalarType, np.complexfloating):
+    print("Demo should only be executed with DOLFINx real mode")
+    exit(0)
+
 import matplotlib.pylab as plt
+
+import basix
+import basix.ufl_wrapper
```

Comment:
These two imports should maybe be after the next comment block that explains them?